app/settings/tokens/-components/edit-form/template.hbs

```diff
@@ -30,18 +30,17 @@
             </div>
         </form.checkboxes>
     </div>
-
     <DeleteButton
         @delete={{action this.deleteToken}}
         @disabled={{this.disabled}}
+        @hardConfirm={{false}}
```

Comment:
I don't think this is necessary. It should be `false` by default.

Reply:
Ok, that is not the behavior I saw but can take a look.

Reply:
@adlius Ok, this is not strictly required and is now removed.